### ISSUE TITLE
Get Started doesn't work with npm 5

### ISSUE THREAD

**Issue by syclik** [electrode-io/electrode#473]
The steps in [Get Started](https://docs.electrode.io/chapter1/quick-start/get-started.html) doesn't work with npm 5. After downgrading to npm 4.6.1, it works.

Here's the error running `clap dev`:
```

$ clap dev
[10:53:41] xclap version 0.2.8
[10:53:41] Loaded CWD/xclap.js
[10:53:41] Added CWD/~/.bin to PATH
[10:53:41] Process dev's dependency serial array [".remove-log-files",".development-env",".clean.build",".mk-dist-dir"]
[10:53:41] -Execute electrode/.remove-log-files as function
[10:53:41] >Done Execute electrode/.remove-log-files as function (0 ms)
[10:53:41] .Execute electrode/.development-env as function
[10:53:41] >Done Execute electrode/.development-env as function (1 ms)
[10:53:41] -Process electrode/.clean.build serial array [".clean.dist",".clean.dll"]
[10:53:41] ..Execute electrode/.clean.dist as function
[10:53:41] >>Done Execute electrode/.clean.dist as function (7 ms)
[10:53:41] --Execute electrode/.clean.dll as function
[10:53:41] >>Done Execute electrode/.clean.dll as function (2 ms)
[10:53:41] >Done Process electrode/.clean.build serial array [".clean.dist",".clean.dll"] (9 ms)
[10:53:41] .Execute electrode/.mk-dist-dir as function
[10:53:41] >Done Execute electrode/.mk-dist-dir as function (0 ms)
[10:53:41] Done Process dev's dependency serial array [".remove-log-files",".development-env",".clean.build",".mk-dist-dir"] (15 ms)
[10:53:41] Process electrode/dev serial array [".webpack-dev",["wds.dev","server-watch","generate-service-worker"]]
[10:53:41] .Execute electrode/.webpack-dev as function
[10:53:41] >Done Execute electrode/.webpack-dev as function (0 ms)
[10:53:41] -Process electrode/dev.S concurrent array ["wds.dev","server-watch","generate-service-worker"]
[10:53:41] ..Execute electrode/wds.dev webpack-dev-server --config ~/node_modules/electrode-archetype-react-app-dev/config/webpack/webpack.config.dev.js --progress --colors --port 2992 --host localhost
[10:53:41] --Execute electrode/generate-service-worker as function
[10:53:41] ..Process server-watch's dependency serial array [".init-bundle.valid.log"]
[10:53:41] >>Done Execute electrode/generate-service-worker as function (1 ms)
[10:53:41] ---Execute electrode/.init-bundle.valid.log as function
[10:53:41] >>>Done Execute electrode/.init-bundle.valid.log as function (1 ms)
[10:53:41] >>Done Process server-watch's dependency serial array [".init-bundle.valid.log"] (1 ms)
[10:53:41] ..Execute electrode/server-watch as function
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: .etmp/bundle.valid.log /Users/daniel/dev/electrode-app/src/server/**/* /Users/daniel/dev/electrode-app/config/**/*
info: No custom webpack config webpack.config.dev.js found in dirs CWD; CWD/archetype/config/webpack
info: Entry point configuration /Users/daniel/dev/electrode-app/src/client/entry.config.js is not found, using default entry point...
Just FYI: optional module not found; Path "/Users/daniel/dev/electrode-app/config/sw-config.js"
Just FYI: optional module not found; Path "/Users/daniel/dev/electrode-app/config/default.json"
info: Not disabling NodeSourcePlugin.  NODE_ENV: 'development'
 10% building modules 1/1 modules 0 active                                         
Project is running at http://localhost:2992/
webpack output is served from http://localhost:2992/js/
webpack bundle is now VALID but has ERRORS                                              
webpack report is served from http://localhost:2992/reporter
```

When I look at http://localhost:2992/reporter, I see:
![screenshot 2017-07-12 11 03 11](https://user-images.githubusercontent.com/425751/28132739-b5c3da50-670b-11e7-8b32-9aef67e80dbe.png)

My computer info:
- MacOS Sierra 10.12.15
- node v6.11.1
- npm v5.2.0 does not work
- npm v4.6.1 works

If this is reproducible, my suggestion would be to document that npm5 doesn't work with electrode in the Get Started section or the requirements section, or find a way to prevent use of npm5 (I don't know if that's easy). If you'd like a pull request with doc change, please let me know and I'll put one together.

(A big thank you to @dustinaleksiuk on Gitter for helping me figure this out!)

**Comment by animesh10** [comment]
Thanks for reporting the issue. I will add supported versions to the documentation, until we support npm5.

**Comment by jchip** [comment]
yes, npm@5 is making a lot of rapid releases and many of them have issues.  Until they stabilizes I think this will continue to occur.  

They just released 5.3.0 that's working again.

Note that webpack-dev-server also just released 2.6.0 a few hours back and it's breaking.  Until we figure what's going on, the work around is do this manually:

```bash
rm -rf node_modules/webpack-dev-server
npm i webpack-dev-server@2.5.1
```

and use lock file to lock it to version 2.5.1.

**Comment by syclik** [comment]
Thanks!
